src/main/js/dynamic/loaded/audit/Details.js

```diff
@@ -43,20 +43,26 @@ class Details extends Component {
     expandComponent(row) {
         let errorMessage = null;
         if (row.errorMessage) {
-            errorMessage = <TextInput label="Error" readOnly name="errorMessage" value={row.errorMessage} />;
+            errorMessage = <TextInput id="auditDetailErrorMessage" label="Error" readOnly name="errorMessage"
+                                      value={row.errorMessage} />;
         }
         let errorStackTrace = null;
         if (row.errorStackTrace) {
-            errorStackTrace = <TextArea inputClass="auditJobDetails" sizeClass="col-sm-12" label="Stack Trace" readOnly name="errorStackTrace"
-                                        value={row.errorStackTrace} cols={"auto"} />;
+            errorStackTrace =
+                <TextArea id="auditDetailStackTrace" inputClass="auditJobDetails" sizeClass="col-sm-12"
+                          label="Stack Trace" readOnly
+                          name="errorStackTrace"
+                          value={row.errorStackTrace} cols={'auto'} />;
         }
 
         return (<div className="auditJobDetails">{errorMessage}{errorStackTrace}</div>);
     }
 
     resendButton(cell, row) {
-        return (<RefreshTableCellFormatter handleButtonClicked={this.onResendClick} currentRowSelected={row}
-                                           buttonText="Re-send" />);
+        return (<RefreshTableCellFormatter
+            id={`audit-detail-refresh-cell`}
```

Comment:
Are the curly braces and back-ticks necessary here?

Reply:
good catch no they are not